src/app/profile-page/profile-page.component.ts

```diff
@@ -99,7 +111,17 @@ export class ProfilePageComponent implements OnInit {
       getRemoteDataPayload(),
       tap((user: EPerson) => this.currentUser = user)
     );
-    this.groupsRD$ = this.user$.pipe(switchMap((user: EPerson) => user.groups));
+    this.groupsRD$ = this.paginationService.getCurrentPagination(this.optionsGroupsPagination.id, this.optionsGroupsPagination).pipe(
+      switchMap((pageOptions: PaginationComponentOptions) => {
+        return this.epersonService.findById(this.currentUser.id, true, true, followLink('groups',{findListOptions: {
+            elementsPerPage: pageOptions.pageSize,
+            currentPage: pageOptions.currentPage
+          }}));
+      }),
+      getAllSucceededRemoteData(),
```

Comment:
it would be good to use getAllCompletedRemoteData here, and deal with the case when the request fails as well (e.g. show an error notification), or even better: don't filter on RemoteData status at all to deal with the case where it is loading too (e.g. show a loading component)

Reply:
@artlowel  I changed getAllSucceededRemoteData for getAllCompletedRemoteData function

Reply:
Thanks!